packages/agreement-process/src/resources/templates/documents/agreementContractTemplate.html

```diff
@@ -125,32 +125,20 @@ <h2>Registrazione</h2>
       <div>
         In data <strong>{{submissionDate}}</strong> alle ore
         <strong>{{submissionTime}}</strong>, l’Infrastruttura ha trasmesso
-        all’erogatore la richiesta di fruizione.
+        {{#if producerDelegationId}} al delegante all’erogazione{{else}} all’erogatore{{/if}} la richiesta di fruizione.
       </div>
-      {{#if producerDelegationId}}
       <div>
         In data <strong>{{activationDate}}</strong> alle ore
         <strong>{{activationTime}}</strong>, l’infrastruttura ha ricevuto
         la dichiarazione di accettazione della richiesta di fruizione
-        (di seguito “dichiarazione di accettazione”) inviata dal delegato all’erogazione,
+        (di seguito “dichiarazione di accettazione”) inviata {{#if producerDelegationId}}dal delegato all’erogazione,{{else}}inviata dall’erogatore,{{/if}}
         tramite l’operatore amministrativo con identificativo Area Riservata
-        <strong>{{producerDelegateId}}</strong>.
-      </div>
-      {{else}}
-      <div>
-        In data <strong>{{activationDate}}</strong> alle ore
-        <strong>{{activationTime}}</strong>, l’Infrastruttura ha ricevuto la
-        dichiarazione di accettazione della richiesta di fruizione (di seguito
-        “dichiarazione di accettazione”) inviata dall’erogatore, tramite
-        l’operatore amministrativo con identificativo Area Riservata
-        <strong>{{activatorId}}</strong>.
+        {{#if producerDelegationId}}<strong>{{activatorId}}</strong>.{{else}}<strong>{{activatorId}}</strong>.{{/if}}
```

Comment:
No need for an if statement, we always display `activatorId` in both cases

```suggestion
        <strong>{{activatorId}}</strong>.
```

Reply:
Done with [1e70bca](https://github.com/pagopa/interop-be-monorepo/pull/1243/commits/1e70bca9ad05d8f2bf848b8d35fd214aa39f7dad)